boards/deprecated.cmake

```diff
@@ -9,3 +9,209 @@
 # set(<old_board_name>_DEPRECATED <new_board_name>)
 
 set(esp32_DEPRECATED esp32_devkitc_wrover)
```

Comment:
shouldn't this also be updated, will it resolve doing 2 sets of deprecations?

Reply:
Yes, I'll consider mapping both `esp32_DEPRECATED` and `esp32_devkitc_wrover_DEPRECATED` to a new name.

Reply:
Done.